src/main/java/org/wso2/maven/humantask/artifact/HumanTaskPluginConstants.java

```diff
@@ -0,0 +1,17 @@
+package org.wso2.maven.humantask.artifact;
```

Comment:
Thanks. Added license header in https://github.com/wso2/maven-tooling-bps/pull/8